providers/openstack/alpha/1-29/cluster-addon-values.yaml

```diff
@@ -0,0 +1,5 @@
+values: |
+  metrics-server:
+    commonLabels:
+      domain: "{{ .Cluster.spec.controlPlaneEndpoint.host }}"
+      clusterAddonVersion: "v2"
```

Comment:
here you hardcode it

Reply:
Hi @janiskemper, it looks like that templating doesn't work here, see https://github.com/SovereignCloudStack/csctl/issues/116

Reply:
Ohh I didn't realize that this was in cluster-addon-values.yaml. I believe that this is not included in the templating. We just didn't have the use case until now.

Reply:
Is the issue that I created in `csctl` still valid then?

Reply:
yes it is. Would you be able to do a PR? I fear that my team won't be able to react in the next days quickly

Reply:
Yes, here it is https://github.com/SovereignCloudStack/csctl/pull/130

Reply:
That was quick :smile: